plugins/search/src/components/SearchModal/SearchModal.test.tsx

```diff
@@ -133,7 +133,7 @@ describe('SearchModal', () => {
       },
     );
 
-    expect(query).toHaveBeenCalledTimes(1);
+    expect(query).toHaveBeenCalledTimes(2);
```

Comment:
same comment as above

Reply:
The local context state was created even when we used the parent, so I extracted it to a separate component, so now it is only called once.